src/lib/i18n.js

```diff
@@ -20,8 +20,18 @@ const init = function I18nInit (lang) {
   }
 }
 
+const setLang = function I18nSet (lang) {
```

Comment:
It was due to the `setLocale` already used in `index.js`, but I renamed it `i18nSetLocale` for better understanding here.